### PR TITLE
WPScan February CVEs

### PR DESCRIPTION
**Pull request by ethicalhack3r**
CVE-2021-24150 to CVE-2021-24168

First time using an automated script to generate the JSON.

### CODE REVIEW

File: 2021/24xxx/CVE-2021-24154.json [hunk 1]
```diff
@@ -1,18 +1,68 @@
 {
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24154.json:1]: [] is too short

File: 2021/24xxx/CVE-2021-24163.json [hunk 2]
```diff
@@ -1,18 +1,73 @@
 {
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24163.json:1]: [] is too short

File: 2021/24xxx/CVE-2021-24164.json [hunk 3]
```diff
@@ -1,18 +1,73 @@
 {
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24164.json:1]: [] is too short

File: 2021/24xxx/CVE-2021-24167.json [hunk 4]
```diff
@@ -1,18 +1,68 @@
 {
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24167.json:1]: [] is too short

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by ethicalhack3r** [comment]
This should now be ready to merge

File: 2021/24xxx/CVE-2021-24153.json [hunk 5]
```diff
@@ -1,18 +1,85 @@
...
+          }
+        }
+      ]
+    }
+  },
+  "description": {
+    "description_data": [
+      {
+        "lang": "eng",
+        "value": "The changelog reads: \"Fixes a stored XSS issue in the Yoast SEO metabox. Thanks Hammad Shamsi for reporting and responsibly disclosing this issue.\""
```
Comment (cristina479) [inline 2021/24xxx/CVE-2021-24153.json]: Should remove "The changelog reads:" and the discoverer of the vulnerability. CVE policy does not give credit to discoverers in the CVE description. Therefore, "hanks Hammad Shamsi for reporting and responsibly disclosing this issue." must be removed.

File: 2021/24xxx/CVE-2021-24160.json [hunk 6]
```diff
@@ -1,18 +1,92 @@
...
+          }
+        }
+      ]
+    }
+  },
+  "description": {
+    "description_data": [
+      {
+        "lang": "eng",
+        "value": "\"A subscriber could upload zip archives containing malicious PHP files that would get extracted to the /rmp-menu// directory. These files could then be accessed via the front end of the site to trigger remote code execution and ultimately allow an attacker to execute commands to further infect a WordPress site.\""
```
Comment (cristina479) [inline 2021/24xxx/CVE-2021-24160.json]: Remove \" ... \" characters

File: 2021/24xxx/CVE-2021-24160.json [hunk 7]
```diff
@@ -1,18 +1,92 @@
...
+                "product_name": "",
+                "version": {
+                  "version_data": [
+                    {
+                      "version_affected": "<",
+                      "version_name": "4.0.4",
+                      "version_value": "4.0.4"
+                    }
+                  ]
+                }
```
Comment (cristina479) [inline 2021/24xxx/CVE-2021-24160.json]: Should be removed if no product is provided

File: 2021/24xxx/CVE-2021-24160.json (hunk 6 above)
Comment (cristina479) [reply 2021/24xxx/CVE-2021-24160.json]: The description must, at least, have the product name.

File: 2021/24xxx/CVE-2021-24161.json [hunk 8]
```diff
@@ -1,18 +1,92 @@
...
+                "product_name": "",
+                "version": {
+                  "version_data": [
+                    {
+                      "version_affected": "<",
+                      "version_name": "4.0.4",
+                      "version_value": "4.0.4"
+                    }
+                  ]
+                }
```
Comment (cristina479) [inline 2021/24xxx/CVE-2021-24161.json]: Should be removed if no product is provided

File: 2021/24xxx/CVE-2021-24161.json [hunk 9]
```diff
@@ -1,18 +1,92 @@
...
+          }
+        }
+      ]
+    }
+  },
+  "description": {
+    "description_data": [
+      {
+        "lang": "eng",
+        "value": "\"Attackers could craft a request and trick an administrator into uploading a zip archive containing malicious PHP files. The attacker could then access those files to achieve remote code execution and further infect the targeted site.\""
```
Comment (cristina479) [inline 2021/24xxx/CVE-2021-24161.json]: The description must, at least, have the product name, and “\” … “\” characters should be removed.

File: 2021/24xxx/CVE-2021-24162.json [hunk 10]
```diff
@@ -1,18 +1,92 @@
...
+                "product_name": "",
+                "version": {
+                  "version_data": [
+                    {
+                      "version_affected": "<",
+                      "version_name": "4.0.4",
+                      "version_value": "4.0.4"
+                    }
+                  ]
+                }
```
Comment (cristina479) [inline 2021/24xxx/CVE-2021-24162.json]: Should be removed if no product is provided

File: 2021/24xxx/CVE-2021-24162.json [hunk 11]
```diff
@@ -1,18 +1,92 @@
...
+          }
+        }
+      ]
+    }
+  },
+  "description": {
+    "description_data": [
+      {
+        "lang": "eng",
+        "value": "\"Attackers could craft a request and trick an administrator into importing all new settings. These settings could be modified to include malicious JavaScript, therefore allowing an attacker to inject payloads that could aid in further infection of the site.\""
```
Comment (cristina479) [inline 2021/24xxx/CVE-2021-24162.json]: The description must, at least, have the product name, and “\” … “\” characters should be removed.

File: 2021/24xxx/CVE-2021-24164.json [hunk 12]
```diff
@@ -1,18 +1,80 @@
...
+          }
+        }
+      ]
+    }
+  },
+  "description": {
+    "description_data": [
+      {
+        "lang": "eng",
+        "value": "Low-level users, such as subscribers, were able to trigger the action, wp_ajax_nf_oauth, and retrieve the connection url needed to establish a connection. They could also retrieve the client_id for an already established OAuth connection."
```
Comment (cristina479) [inline 2021/24xxx/CVE-2021-24164.json]: The description must, at least, have the product name

File: 2021/24xxx/CVE-2021-24165.json [hunk 13]
```diff
@@ -1,18 +1,80 @@
...
+          }
+        }
+      ]
+    }
+  },
+  "description": {
+    "description_data": [
+      {
+        "lang": "eng",
+        "value": "The wp_ajax_nf_oauth_connect AJAX action was vulnerable to open redirect due to the use of a user supplied redirect parameter and no protection in place."
```
Comment (cristina479) [inline 2021/24xxx/CVE-2021-24165.json]: The description must, at least, have the product name

**Pull request merged**